jbot/src/main/java/me/ramswaroop/jbot/core/slack/models/Event.java

```diff
@@ -62,6 +65,8 @@
     private String domain;
     @JsonProperty("email_domain")
     private String emailDomain;
+    @JsonProperty("team")
+    private String teamId;
```

Comment:
One last question before I merge, you get `team` in the json response but you have named the variable as teamId, is it because I have followed the same convention in other models?